ubuntu/14.04/standard/Dockerfile

```diff
@@ -154,10 +154,9 @@ RUN apt-get update \
     python3-pip \
  && rm -rf /var/lib/apt/lists/*
 
-# Install Ruby
+# Install Ruby requirements
 RUN apt-get update \
- && apt-get install -y ruby ruby-dev ruby-bundler \
- && rm -rf /var/lib/apt/lists/*
+ && apt-get install -y libz-dev openssl libssl-dev
```

Comment:
These are necessary for the Ruby installation.  They were on the machine that was used to build the Ruby installation (these libraries were required for Ruby to access gem using https).  They are added here so that the Ruby installation in the tools cache will have access to them.